specification/hugr.md

```diff
@@ -315,30 +315,29 @@ and an Output node, whose outputs and inputs respectively match the inputs and
 outputs of the containing DFG.
 
 ```mermaid
-stateDiagram-v2
-    [*] --> DFG0
-    [*] --> DFG0
-
-    state DFG0 {
+flowchart
```

Comment:
I guess the style here is changed for consistency, but it doesn't look as good to my eye. The `flowchart` style seems to have a tendency to cross wires which makes it a bit harder to read

Reply:
Yes, I found it was impossible to draw non-local edges with the `stateDiagram-v2` structure.